docker/README.md

```diff
@@ -100,17 +156,27 @@ Configuration](#advanced-configuration)**
 
 ### Run Supabase and StudyU
 
-Make sure that you thoughtfully completed all the previous steps and then start
-the following services:
+Make sure that you thoughtfully completed all the previous steps. You can either
+use the StudyU CLI or manually run the docker-compose commands.
+
+StudyU CLI:
```

Comment:
My linter would say that this is a level 4 heading🙊😜 same as the ones below

Reply:
I tried that but then the linter said no equal headings in one file haha. I can enumerate the headings if you think that'd be a better idea. Nevertheless, we should move the documentation [to the website](https://www.studyu.health/docs/development/selfhosting) at some point to split it up and increase readability

Reply:
Oh true! I'd be good with turning off the rule for no equal headings, I don't like that rule. You can decide if you want to turn it off or leave it as is :)